tools/style/eclipse.importorder

```diff
@@ -0,0 +1,23 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#Organize Import Order
+0=org.apache.seatunnel
+1=org.apache
+2=java
+3=javax
+4=org
+5=com
```

Comment:
Can scala be added?
And change the file name to `seatunnel.importorder`

Reply:
Thanks for the suggestions. I will take a look and see whether we could add scala into it.